src/screens/CardInfo/CardInfoScreen.tsx

```diff
@@ -0,0 +1,216 @@
+            <Text style={styles.text}>Benefits</Text>
+          </Tile>
+        </Pressable>
+        {/*
+        <CustomButton onPress={() => alert('Work in progress')} title='Business' type='primary' />
+                        <CustomButton onPress={() => alert('Work in progress')} title='Benefits' /> */}
+      </View>
+      {/* everything below is in temp form, what is rendered will be based on button^ pressed */}
+      {screenState ? (
+        <View style={styles.container}>
+          <BoxContainer style={styles.boxContainer}>
+            <Text style={[styles.text, { paddingBottom: 40 }]}>Business name</Text>
+            <Text style={[styles.text, { paddingBottom: 40 }]}>Address</Text>
+            <Text style={styles.text}>Info</Text>
+          </BoxContainer>
+          {/* todo: apply proper paddings
+                even more temp solutions below */}
+          <View style={styles.claimButton}>
+            <CustomButton onPress={() => alert('Work in progress')} title='Claim Benefits' />
+          </View>
+          <View style={styles.showCard}>
+            <CustomButton onPress={() => alert('Work in progress')} title='Show Card' />
+          </View>
+        </View>
+      ) : (
+        <View style={styles.container}>
+          <View style={styles.benefitsContainer}>
+            <Text style={styles.text}>List of benefits</Text>
+            {/*todo: style the tiles in list */}
+            <BenefitList
+              benefits={benefits}
+              onPress={() => {
+                //to add: passing Benefit object
+                navigation.push('BenefitDescriptionScreen');
+              }}
+              //backgroundColor based on affordability
+              customBenefitTileStyle={{ width: '100%', height: 60 }}
+            />
+          </View>
+          <View style={[styles.buttonsContainer, { marginTop: '10%' }]}>
+            <CustomButton
+              onPress={() => alert('Work in progress')}
+              title='Cancel'
+              customButtonStyle={styles.button}
+            />
+            <CustomButton
+              onPress={() => alert('Work in progress')}
+              title='Save'
+              customButtonStyle={styles.button}
+            />
+          </View>
+        </View>
+      )}
+      <TapBar navigation={navigation} />
+    </View>
+  ) : (
+    <View style={StyleBase.container}>
+      <TopBar iconLeft={'arrow-left'} onPressLeft={() => onPressBack(navigation)} />
+      <Text>This is real card info</Text>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  accountTileContainer: {
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    gap: 100,
+  },
+  benefitsContainer: {
+    width: '90%',
+    height: '70%',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    marginTop: '2.5%',
+    borderColor: '#000',
+    borderWidth: 1,
+  },
+  boxContainer: {
+    height: '53.75%',
+    width: '89.72%',
```

Comment:
Boje się, że z tymi wartościami, które ustawiamy (procentowe) będzie kłopot przy ustawieniu dobrych responsywnych widoków. Wydaje się to dzikie.

Reply:
bralem wartosci z figmy - przeliczalem piksele na procent calego ekranu xd

Reply:
jesteś crazy haha

Reply:
tylko w takim wypadku trzeba napisać hooki, które zgarniają wartości wysokość/szerokość na ekranie i wtedy je przeliczać w ten sposób. Bo te 53.75% może być u Ciebie odpowiednią szerokością, a u mnie na innym modelu już nie. Czy się mylę?